### PR TITLE
Get element ID by path

### PR DESCRIPTION
**Pull request by lukmzig**
## Changes in this pull request
Part of https://github.com/pimcore/studio-ui-bundle/issues/346

### CODE REVIEW

File: src/DependencyInjection/PimcoreStudioBackendExtension.php [hunk 1]
```diff
@@ -77,6 +77,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('updater.yaml');
         $loader->load('users.yaml');
         $loader->load('versions.yaml');
+        $loader->load('elements.yaml');
```
Comment (mattamon) [inline src/DependencyInjection/PimcoreStudioBackendExtension.php]: Do you even alphabet 🔤 :D

File: src/Element/Request/ElementPathParameter.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+namespace Pimcore\Bundle\StudioBackendBundle\Element\Request;
+
+/**
+ * @internal
+ */
+final readonly class ElementPathParameter
+{
+    public function __construct(
+        private string $elementPath
```
Comment (mattamon) [inline src/Element/Request/ElementPathParameter.php]: Could we just name it path?
Wdyt? It is a little redundant in the moment. :)

Comment (lukmzig) [reply src/Element/Request/ElementPathParameter.php]: sure, we can rename it to path :)

File: src/DependencyInjection/PimcoreStudioBackendExtension.php (hunk 1 above)
Comment (lukmzig) [reply src/DependencyInjection/PimcoreStudioBackendExtension.php]: abcduuve

File: src/OpenApi/Config/Tags.php [hunk 3]
```diff
@@ -79,12 +83,13 @@ enum Tags: string
...
     case DataObjects = 'DataObjects';
     case Dependencies = 'Dependencies';
+    case Elements = 'Elements';
     case Notes = 'Notes';
     case Properties = 'Properties';
     case Schedule = 'Schedule';
     case Settings = 'Settings';
     case Translation = 'Translation';
-    case Thumbnails = 'Thumbnails';
+    case AssetThumbnails = 'Asset Thumbnails';
```
Comment (mattamon) [inline src/OpenApi/Config/Tags.php]: 🔡

**Review by mattamon — APPROVED** [review]
(>‿◠)✌

**Pull request merged**